A3A/addons/core/functions/Base/fn_checkWinCondition.sqf

```diff
@@ -0,0 +1,105 @@
+#include "..\..\script_component.hpp"
+FIX_LINE_NUMBERS()
+
+private _factionMoney = server getVariable ["resourcesFIA",0];
+private _hr = server getVariable ["hr",0];
+private _victoryZones = airportsX + milbases + outposts + resourcesX + factories + seaports;
+private _victoryZonesLogistical = airportsX + milbases + seaports;
+private _popTotal = 0;
+private _popKilled = 0;
+private _missingMoney = ((2000000 - _factionMoney) call BIS_fnc_numberText) splitString " " joinString ",";
+private _popReb = 0;
+private _popGov = 0;
+private _popMajority = 0;
+
+{
+    private _city = _x;
+    private _cityData = server getVariable _city;
+    _cityData params ["_numCiv", "_numVeh", "_supportGov", "_supportReb"];
+
+    _popTotal = _popTotal + _numCiv;
+    if (_city in destroyedSites) then { _popKilled = _popKilled + _numCiv; continue };
+
+    _popReb = _popReb + (_numCiv * (_supportReb / 100));
+    _popGov = _popGov + (_numCiv * (_supportGov / 100));
+} forEach citiesX;
+
+_popMajority = _popTotal * 0.75;
+
+private _resourcesTotal = count (resourcesX);
+private _resourcesOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (resourcesX));
+
+private _factoriesTotal = count (factories);
+private _factoriesOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (factories));
+
+private _outpostsTotal = count (outposts);
+private _outpostsOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (outposts));
+
+private _seaportsTotal = count (seaports);
+private _seaportsOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (seaports));
+
+private _milbasesTotal = count (milbases);
+private _milbasesOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (milbases));
+
+private _airportsTotal = count (airportsX);
+private _airportsOwned = ({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (airportsX));
+
+switch (victoryCondition) do
+{
+    //Normal Victory
+    case 0:
+    {
+        if ((_popReb > _popGov) && {({sidesX getVariable [_x,sideUnknown] isEqualTo teamPlayer} count (airportsX + milbases)) isEqualTo count (airportsX + milbases)}) then {
+            isNil {["ended", true] call A3A_fnc_writebackSaveVar};
+            ["End1",true,true,true,true] remoteExec ["BIS_fnc_endMission"];
+        }else{
```

Comment:
A man was given infinite space, and yet he still chose to build a tiny square hut

Use spaces, it's free, infinite and makes it easier to read
```suggestion
        } else {
```